core/lib/spree/event/configuration.rb

```diff
@@ -59,6 +59,13 @@ def adapter
               order.do_something
               Spree::Event.fire 'event_name', order: order
 
+          - You need to register your custom events before firing or subscribing
+            to them (no need for Solidus' custom ones). Example:
```

Comment:
Solidus' native ones?

Reply:
Perhaps "not necessary for events provided by Solidus or other extensions" would be more clear.

Reply:
Fixed, I used @jarednorman's suggestion :+1: